src/configurationProvider.ts

```diff
@@ -0,0 +1,74 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { parse } from 'jsonc-parser';
+import { OmniSharpServer } from './omnisharp/server';
+import * as serverUtils from './omnisharp/utils';
+import { AssetGenerator, addTasksJsonIfNecessary, createLaunchConfiguration, createAttachConfiguration, containsDotNetCoreProjects} from './assets';
+import { WorkspaceInformationResponse } from './omnisharp/protocol';
+
+export class CSharpConfigurationProvider implements vscode.DebugConfigurationProvider {
+    private server: OmniSharpServer;
+
+    public constructor(server: OmniSharpServer) {
+        this.server = server;
+    }
+
+    /**
+     * TODO: Remove function. 
```

Comment:
>TODO [](start = 7, length = 4)

Can we remove this now?

Reply:
Can I retrieve other workspace folder information with the change in #1806?

Reply:
@DustinCampbell should this work now?

Reply:
You're waiting on https://github.com/OmniSharp/omnisharp-roslyn/issues/909. OmniSharp is not being launched on multiple workspace folders. Instead, the user can now launch OmniSharp on targets within multiple workspace folders.

Reply:
In that case I would suggest:
* Add a link the bug
* Instead of checking if the solution path is the same, seems like we should be checking if the folder is what we launched OmniSharp against.

Reply:
Yes, I hadn't spotted that we were checking the SolutionPath against the folder name. That's going to be wrong. Note that you *can* have an `.sln` file, and in that case SolutionPath will point to the `.sln`.